src/Autocomplete/src/EntityAutocompleterInterface.php

```diff
@@ -46,4 +46,9 @@ public function getValue(object $entity): mixed;
      * Note: if SecurityBundle is not installed, this will not be called.
      */
     public function isGranted(Security $security): bool;
+
+    /**
+     * Return Form option by name.
+     */
+    public function getFormOption(string $name): mixed;
```

Comment:
I think the new method should be `getGroupBy()`. This interface is meant to be usable in custom classes that have nothing to do with a for.